frontend/src/app/modules/home/issues/issues.component.html

```diff
@@ -34,15 +34,15 @@
                 </ng-template>
                 <ng-template pTemplate="header">
                     <tr>
-                        <th class="name" pSortableColumn="name">
+                        <th class="name" pSortableColumn="newest.occurredOn">
```

Comment:
Why did you change the sort field in the first column?

Reply:
Because it useless to sort by name, better to sort from newest issue to oldest.